xscen/io.py

```diff
@@ -518,6 +492,30 @@ def coerce_attrs(attrs):
             raise
 
 
+def _rechunk_for_saving(ds, rechunk):
```

Comment:
I think you need to add it to the `__all__` at the begining of the file to be able to import it correctly in the tests.

Reply:
I would also consider removing the "_", as users might want to use this function directly ?

Reply:
All done, thank you!